core/Tracker/Response.php

```diff
@@ -25,7 +25,7 @@ public function init(Tracker $tracker)
     {
         ob_start(); // we use ob_start only because of Common::printDebug, we should actually not really use ob_start
 
-        if ($tracker->isDebugModeEnabled() && TrackerConfig::getConfigValue('enable_sql_profiler')) {
+        if ($tracker->isDebugModeEnabled() && TrackerConfig::getConfigValue('enable_sql_profiler', $this->getIdSite())) {
```

Comment:
fyi @diosmosis this method getIdSite doesn't seem to be defined?

Reply:
we could just remove idSite

Reply:
:+1: this one doesn't need to be overridden